### ISSUE TITLE
[Question] Source code of script

### ISSUE THREAD

**Issue by MichaelDeBoey** [Allen1212/trace-ark-trades#3]
Hi @Allen1212!

I was wondering if we somehow could have a look at the source code that's driving this repo?

I was also wondering if you still have the holdings data PDF/CSVs stored somewhere?
Or are they only used to process the data and push it to this repo and removed afterwards?

**Comment by Allen1212** [comment]
Hey Michaël,

I am thinking if I need to push the source code to the repo because most people using this repo do not have a programming background. I don't wanna confuse them. I hope what they need to do are COPY and PASTE. In addition, I finish the coding in one night using Python. I didn't comment/explain code enough, which may cost people's time to reading the code😅

Yes, I maintain the historical trading data! After handling the daily trade data from CSVs (https://ark-funds.com/investor-resources), the processed data will be stored. But I only use these data to generate the pinescripts and did not push the original data to the repo.

Thanks for your comment!

**Comment by MichaelDeBoey** [comment]
@Allen1212 I understand that putting the source code in this repo could confuse some people, but maybe you can put it in another repo?
I think it can be valuable for people to learn from, even if it isn't perfect.

People can even clean up the code for you or make it more performant if they want by just creating a PR then.

If possible, you can maybe also create a repo where you put the original CSVs, so people could use them too for their own scripts if wanted?

**Comment by MichaelDeBoey** [comment]
Ping @Allen1212

**Comment by jingzact** [comment]
@MichaelDeBoey I have been following this repo since the data was helpful to get started on my own project also for reconciliation purposes. I have my own code to download the daily positions now. Are you just trying to get the daily data?

**Comment by MichaelDeBoey** [comment]
@jingzact I also trying to learn from how things are done here

**Comment by Allen1212** [comment]
Sorry @MichaelDeBoey, I am pretty busy recently and forgot to reply to your last question😳 Thanks for your suggestion. I will consider putting the source code or raw data to another repo. When I get it done, I will let you know😜

**Comment by Allen1212** [comment]
@jingzact I am glad this repo could help your project🥂

**Comment by jingzact** [comment]
@MichaelDeBoey I have created a dashboard (data visualization) in Python for the ARK funds mainly for proof of concept and educational purposes. 

I extracted the historical positions from the pinescripts provided here, but am not using the actual pinescripts. I also plan to open source that dashboard once I have time to set up the repo. In the meantime, If you are interested, the dashboard can be found here: www.ark-board.com

Many thanks to this repo and @Allen1212 who provided the historical data back to Nov 2019!
